hdfs_namenode/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG - hdfs_namenode
 
+1.1.1 / Unreleased
```

Comment:
Since we adding support for instance tags it's a feature and not a bug fix: version should be `1.2.0`